package.json

```
@@ -24,7 +24,7 @@
     "@typescript-eslint/parser": "^2.0.0",
     "eslint-plugin-typescript": "^0.14.0",
     "fastify": "^2.11.0",
-    "iltorb": "^2.3.0",
+    "iltorb": "2.3.0",
```

[2.4.3](https://github.com/nstepien/iltorb/blob/master/CHANGELOG.md#243---2019-04-24) should support node 12 and 6

```suggestion
    "iltorb": "2.4.3",
```